lib/Graph/Graph.cpp

```diff
@@ -177,9 +180,15 @@ class AbstractDottyPrinter {
   }
 
   std::string uniqueVertexName(void *N) {
+    VertexNumberMap::iterator i;
+    bool inserted;
+    std::tie(i, inserted) = vertex_numbers.insert(std::make_pair(N, 0u));
+    if (inserted)
```

Comment:
We usually use `{}` even in such single-line cases. It is described somewhere in FB-style.